### ISSUE TITLE
It tries to auto-update itself, fail, then close

### ISSUE THREAD

**Issue by mickoz** [mzomparelli/zVirtualDesktop#216]
When opening zVirtualDesktop, it says there is an update, then ask if we want to install it, then without choosing any choice it tries to auto-update itself, fail, then the program close.  Always doing this prevent to use it at all since it always auto-close.  Quick work-around is to disable internet so it does not find the update, but re-enabling internet will eventually lead to it trying to do the same thing.

I tried to run it as administrator instead, and the auto-update that apply (without we even click yes or no) does not fail, but it then try to auto-update constantly, so always starting, auto-update, restarting, auto-update, etc.

What I expect as behavior is for it to ask if there is an update, and actually wait for our YES or NO answer.  Not auto-updating.

**Comment by mickoz** [comment]
Trial or not, why does it tries to auto-update constantly in an endless loop?  Does not seem like the correct behavior.

**Comment by mzomparelli** [comment]
Because you're expired.

**Comment by mzomparelli** [comment]
You make enough to buy it and you'd be supporting a single father. Instead you chose to violate my software agreement.

**Comment by mickoz** [comment]
Does not make sense that it tries to auto-update when expired, getting into an infinite loop.

**Comment by mzomparelli** [comment]
You're trial has expired? Proper users never have this issue. It's just you.

**Comment by mzomparelli** [comment]
I'll say it again....you have violated my software agrement

**Comment by mzomparelli** [comment]
If you want to buy a license then you must contact me via email. You will not be getting the list price.

**Comment by mzomparelli** [comment]
I know a lot about you Michael Muryn from Quebec, Canada. Twitter, Facebook, etc. Home address.

**Comment by mzomparelli** [comment]
I really don't care if you steal 1.0.89 anymore. You are locked out from 1.0.90 which has several bug fixes that you will never see. Including the activation one.

**Comment by mzomparelli** [comment]
Is $25 too much for you. You have a good job so I don't think it is. You obviously want to use the program. What's the problem Michael?

**Comment by mickoz** [comment]
I don't have a job BTW, you are far from knowing my actual situation unfortunately and it is not sunny at all... would only discuss in private of this.  Feel free to email me since you have all my info.

That being said, why not simply say "Trial expired" instead of auto-updating?

And if you spy on your users, that is a bit concerning what your program can actually do.  Is it a trojan, eh? ;-)

PS. Won't lie, I have done like *you* suggested in another post when you decided to make it closed source, using a decompiler.  Still, program should say "Trial expired" and not auto-update constantly.  Unless you compiled an auto-updater endless loop just for me!

**Comment by mzomparelli** [comment]
The program will say trial expired for people not hacking Michael. You have been trying to hack zVD. You have tried editing the local license and bypassing internet activation.

**Comment by mzomparelli** [comment]
Update to the new version 1.0.90 and I'll give you another trial. If you hack again you will have a permanent ban.

**Comment by mickoz** [comment]
Hmmm, if you read what I said above, I runned the program as administrator, the update SUCCEEDED, I saw the .exe being updated (to latest version I suppose), and it then started and was auto-updating in an infinite loop.

**Comment by mzomparelli** [comment]
You did run 1.0.91, but now you are back to 1.0.89. I can't help you if you are running 1.0.89.

**Comment by mzomparelli** [comment]
I collect certain info that can help me identify users trying to bypass my activation. It's sad that I have to do that because I don't want to, however, there are people like you that try to steal software.

**Comment by mzomparelli** [comment]
Please wait while I extend your trial

**Comment by mzomparelli** [comment]
You should be good now. Please keep in mind that I'm not a company. I'm a single father trying to make my way. Please help me out.

**Comment by mzomparelli** [comment]
I'm sorry, I didn't know you didn't have a job. It can be tough out there. You clearly have skills though. Forgive me for the way I have spoken and please be supportive of my work.

**Comment by mickoz** [comment]
BTW, when going from expired trial to trial, we need to re-enable mouse hook and keyboard hook.  Probably because you disable those when a trial is expired.  Actually I think I only re-enabled keyboard hook in my case.

**Comment by mzomparelli** [comment]
Yes, that is true. Sorry about that. When you activate an expired trial then it will auto enable those. It will not for trial extensions.

**Comment by mickoz** [comment]
Not having a job is not the worst thing, even if financially it does hurt my family (father of a young 2 y/o here...).  I will look into contacting you and chat some more if you don't mind, would have a couple question about your software, etc.  What email you prefer I use for this?

**Comment by mzomparelli** [comment]
Yes, please contact me. Maybe we can work something out. I really am a nice guy. I was mad at you for your actions. However, as I talk to you here, you seem like a genuinely nice guy.

zVirtualDesktop@gmail.com
